ProductionSystem/Service/ProductionManagerHandler.py

```diff
@@ -17,11 +17,15 @@
 transTypes = [six.string_types, int, list]
 
 
-class ProductionManagerHandlerBase(RequestHandler):
+class ProductionManagerHandler(RequestHandler):
 
-  def setDatabase(self, oDatabase):
-    global database
-    database = oDatabase
+  @classmethod
+  def initializeHandler(cls, serviceInfoDict):
+    """ Initialization of DB object
+    """
+
+    cls.gProductionDB = ProductionDB()
```

Comment:
I do not think that we should prepend "g" to the name of the database here. It used to denote global objects. Now this is a class object which is in the code clearly seen as always used as cls.productionDB

Reply:
I changed all.